examples/nodejs/stock.js

```diff
@@ -20,10 +20,11 @@ module.exports = function (context, callback) {
 
             try {
                 var parsed = JSON.parse(resp.slice(3));
-                var lastTrade = parsed[0]['l_cur']
-                callback(200, `{ "text": "${symbol} last traded at ${lastTrade}" }`);
+                var lastTrade = parsed[0]['l_cur'];
+                var headers = {'Content-Type': 'application/json'}; 
```

Comment:
Thanks, good idea to add this!  But since it's used in the catch block, it should be moved above the `try` on line 21, I think.